src/project/build.scala

```diff
@@ -35,7 +35,7 @@ object Extras {
     // should include these things in the distributed jar
     "io.spray"         %% "spray-json"  % "1.3.2",
     "com.microsoft.CNTK" % "cntk_jni"   % "2.0rc3",
-    "org.opencv"         % "opencv_jni" % "3.2.0"
+    "org.openpnp"         % "opencv" % "3.2.0-1"
```

Comment:
Also, could you please make it use the CNTK jar that Ratan uploaded to Maven:
(http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.microsoft.cntk%22)
and make sure we don't need our Maven repo anymore? That'll completely unblock our move to Spark packages.